docker-compose.yml

```diff
@@ -96,6 +96,7 @@ services:
     environment:
       IIB_DEV: 'true'
       REQUESTS_CA_BUNDLE: /etc/pki/tls/certs/ca-bundle.crt
+      GODEBUG: 'x509ignoreCN=0'
```

Comment:
proper fix for this is to regenerate certificate with SANs instead of CN

Reply:
Yeah, I see. Is it clever to create one for the dev environment? Or everyone should create it is own?

Reply:
removed